### PR TITLE
text: update message and btn radius

### PR DESCRIPTION
**Pull request by utkarsha-deriv**

### CODE REVIEW

File: src/features/Apiexplorer/LoginDialog/index.tsx [hunk 1]
```diff
@@ -34,7 +34,14 @@ export const LoginDialog = ({ setToggleModal }: TLoginDialog) => {
...
           >
-            <div className={styles.modal}>Log in or sign up to continue.</div>
+            <div className={styles.modal}>
+              This API call must be authorised because it requires access to your account
+              information.
+            </div>
+            <div className={styles.modal}>
+              Log in to your Deriv account to proceed. If you don’t have a Deriv account, sign up
+              first.
+            </div>
```
Comment (hubert-deriv) [inline src/features/Apiexplorer/LoginDialog/index.tsx:44]: Perhaps, it is possible to put these 2 sentences in two seperate `<p>` elements? I believe that makes more sense and I think we can then remove the padding from the `.modal` class.

**Review by hubert-deriv — CHANGES_REQUESTED** [review]

File: src/features/Apiexplorer/LoginDialog/index.tsx (hunk 1 above)
Comment (utkarsha-deriv) [reply src/features/Apiexplorer/LoginDialog/index.tsx:44]: I'll change `<div>` to `<p>` but we will still need the padding.

**Review by hubert-deriv — CHANGES_REQUESTED** [review]
Looks good to me. Although, it looks like that CodeCoverage is failing. Could you run tests locally and see what coverage has to be fixed?

File: src/features/Apiexplorer/LoginDialog/index.tsx (hunk 1 above)
Comment (hubert-deriv) [reply src/features/Apiexplorer/LoginDialog/index.tsx]: Sure, that is no problem

**Pull request merged**